pkg/machine/applehv/config.go

```diff
@@ -109,6 +109,18 @@ func (v AppleHVVirtualization) LoadVMByName(name string) (machine.VM, error) {
 	return m.loadFromFile()
 }
 
+// setVMFile creates a new machine file pointing to the given path and assigns
+// it to the destination
+func setVMFile(dest *machine.VMFile, path string) error {
```

Comment:
I'd find it cleaner if we return a `(*machine.VMFile, error)`.  In/out arguments are less idiomatic and hard to brain parse.